docs/index.html

```diff
@@ -4741,7 +4740,7 @@ <h2>Vocabulary</h2>
           "@id": "https://schema.org/unitCode"
         },
         "title": "Measurement Unit",
-        "description": "A string value indicating a Measurement Unit from UN/ECE Recommendation 20, Units of Measure used in International Trade e.g. GRM = gram - see <a href=\"http://www.unece.org/fileadmin/DAM/cefact/recommendations/rec20/rec20_rev3_Annex3e.pdf\">rec20_rev3</a>"
+        "description": "A string value indicating a Measurement Unit from UN/ECE Recommendation 20, Units of Measure used in International Trade, e.g., GRM = gram - see <a href=\"http://www.unece.org/fileadmin/DAM/cefact/recommendations/rec20/rec20_rev3_Annex3e.pdf\">rec20_rev3</a>"
```

Comment:
any of these changes I believe will be overwritten by autogeneration.  the above core template item are fine.  @OR13  I know you commented on somehting similar recently

Reply:
If they're going to be overwritten, then there's no need to pull them from the commit; they'll just be overwritten.

In such case, however, it _would_ be helpful to know where to target my fix, so that future overwrites bring correct punctuation instead of perpetuating the errors.

Reply:
:) sorry about that - Those are sourced from the schemas store in `docs/schemas/`

Reply:
yeah, I think the correct way to handle this would be for us to generate the html, and then run lint on it which would fix line length / indentation.

Reply:
@OR13 - Note that my changes here were not primarily about "line length / indentation" but punctuation, spelling, grammar, and the like. I've now applied my efforts to the `docs/schemas/` files, so the over-writes should now include the fixes I made here.

Reply:
@TallTed thanks for these PRs, we really appreciate them!